GOVERNANCE.md

```diff
@@ -0,0 +1,146 @@
+
+## Project Roles
+
+For governance purposes, there are two formal roles for the Debezium project.
+
+* **Steering Committee**: A committee composed of committers with the final authority over all decisions made on the project.
+* **Committers**: A group of contributors with the right to merge pull requests into the Debezium repositories and publish releases.
+
+### Debezium Committers
+
+A Debezium committer is an individual who has been granted the right to merge pull requests into one of several Debezium GitHub repositories.
+A pull request requires a voting/review procedure, outlined below in the [Voting Rules and Procedures](#voting-rules-and-procedures) section prior to merging.
+
+Debezium's committers are listed in the [COMMITTERS.yml](./committers.yml).
+
+### Steering Committee
+
+The Debezium steering committee has the final authority over all decisions regarding the Debezium project, including:
+
+* Whether pull requests are merged or retained in the event of a dispute.
+* Who should become a committer or member of the steering committee.
+* Whether any changes are necessary to the project's governance or other assets in this repository.
+
+Debezium's steering committee members are listed in the [COMMITTEE.yml](./committee.yml).
+
+## Voting rules and procedures
+
+The Debezium project uses a voting system to guarantee no single member or entity can dominate the project, and ensures continued success and diversity in the project's forward vision.
+
+The formal voting process involves the creation of a GitHub issue or pull request.
+A vote is indicated by specifying `yes` or `no` on the issue or pull request, which can include the thumbs-up or thumbs-down emojis.
+After a defined period of time, votes are tallied, and the outcome is published.
+
+Debezium uses several voting mechanisms and procedures for various purposes.
+
+* [Pull request approvals](#pull-request-approvals)
+* [Lazy consensus votes](#lazy-consensus-votes)
+* [Explicit majority votes](#explicit-majority-votes)
+
+Debezium also relies on the notion of **binding** and **non-binding** votes.
+
+A **binding vote** is one cast by specific individuals with given roles based on the voting procedure, who's vote is used to calculate the outcome.
+A **non-binding vote** is one cast by individuals who do not have binding vote roles, and these votes do not count toward the outcome, but are used to guide those with binding votes to a resolution that best fits all involved.
+
+The following describes for each procedure, those who may vote, how the votes are recorded, and duration of the vote.
+
+### Pull request approvals
+
+Pull requests against most repositories may be merged after receiving at least two positive committer votes.
+Voting for pull requests can be done using a comment or approving the pull request on GitHub.
+If the pull request author is a member of the Debezium steering committee, this counts as one of the two required votes.
```

Comment:
So steering committee members need just one supportive vote by someone else? Not 100% sure how I feel about it, feels a bit like "rules for thee but not for me?" Let's discuss it in the call later today.

Reply:
This is something I borrowed from Strimzi, but I am certainly open to changing it if there are concerns.

Reply:
For now, let's take a lean and mean approach, focusing on consistency across the board.  There will need to be at least two committer/committee reviews, regardless of the author's role.